software/nosql/src/main/java/brooklyn/entity/nosql/riak/RiakNode.java

```diff
@@ -142,6 +140,27 @@
     MethodEffector<Void> LEAVE_RIAK_CLUSTER = new MethodEffector<Void>(RiakNode.class, "leaveCluster");
     MethodEffector<Void> COMMIT_RIAK_CLUSTER = new MethodEffector<Void>(RiakNode.class, "commitCluster");
 
+    AttributeSensor<String> NODE_GET_FSM_TIME_MEAN = Sensors.newStringSensor("node_get_fsm_time_mean", "Time between reception of client read request and subsequent response to client");
+    AttributeSensor<String> NODE_PUT_FSM_TIME_MEAN = Sensors.newStringSensor("node_put_fsm_time_mean", "Time between reception of client write request and subsequent response to client");
+    AttributeSensor<String> OBJECT_COUNTER_MERGE_TIME_MEAN = Sensors.newStringSensor("object_counter_merge_time_mean", "Time it takes to perform an Update Counter operation");
+    AttributeSensor<String> OBJECT_SET_MERGE_TIME_MEAN = Sensors.newStringSensor("object_set_merge_time_mean", "Time it takes to perform an Update Set operation");
+    AttributeSensor<String> OBJECT_MAP_MERGE_TIME_MEAN = Sensors.newStringSensor("object_map_merge_time_mean", "Time it takes to perform an Update Map operation");
+    AttributeSensor<String> SEARCH_QUERY_LATENCY_MEDIAN = Sensors.newStringSensor("search_query_latency_median", "Search query latency");
+    AttributeSensor<String> SEARCH_INDEX_LATENCY_MEDIAN = Sensors.newStringSensor("search_index_latency_median", "Time it takes Search to index a new document");
+    AttributeSensor<String> CONSISTENT_GET_TIME_MEAN = Sensors.newStringSensor("consistent_get_time_mean", "Strongly consistent read latency");
+    AttributeSensor<String> CONSISTENT_PUT_TIME_MEAN = Sensors.newStringSensor("consistent_put_time_mean", "Strongly consistent write latency");
+
+    List<AttributeSensor<String>> ONE_MINUTE_STRING_SENSORS = ImmutableList.of(NODE_GET_FSM_TIME_MEAN, NODE_PUT_FSM_TIME_MEAN,
+            OBJECT_COUNTER_MERGE_TIME_MEAN, OBJECT_SET_MERGE_TIME_MEAN, OBJECT_MAP_MERGE_TIME_MEAN,
+            SEARCH_QUERY_LATENCY_MEDIAN, SEARCH_INDEX_LATENCY_MEDIAN,
+            CONSISTENT_GET_TIME_MEAN, CONSISTENT_PUT_TIME_MEAN);
+
+    ConfigKey<Boolean> DO_OS_TUNING = new BasicConfigKey<Boolean>(
```

Comment:
Use `ConfigKeys.newBooleanConfigKey` here, although this seems to be a duplicate of https://github.com/apache/incubator-brooklyn/blob/master/software/nosql/src/main/java/brooklyn/entity/nosql/riak/RiakNode.java#L49-L50 from #563